erpnext/crm/doctype/lead/lead.json

```diff
@@ -106,21 +136,21 @@
    "bold": 0, 
    "collapsible": 0, 
    "columns": 0, 
-   "fieldname": "company_name", 
-   "fieldtype": "Data", 
+   "fieldname": "gender", 
+   "fieldtype": "Select", 
    "hidden": 0, 
    "ignore_user_permissions": 0, 
    "ignore_xss_filter": 0, 
    "in_filter": 0, 
    "in_global_search": 0, 
-   "in_list_view": 1, 
+   "in_list_view": 0, 
    "in_standard_filter": 0, 
-   "label": "Organization Name", 
+   "label": "Gender", 
    "length": 0, 
    "no_copy": 0, 
-   "oldfieldname": "company_name", 
-   "oldfieldtype": "Data", 
+   "options": "\nMale\nFemale", 
```

Comment:
If we do others, we need to give text box to mention the other gender. Blank field is better I think.